scripts/g.extension/g.extension.py

```diff
@@ -388,16 +398,31 @@ def etree_fromfile(filename):
 
 def etree_fromurl(url):
     """Create XML element tree from a given URL"""
+    base_message = "Download file from <{url}> failed".format(url=url)
```

Comment:
Needs to be translatable too if you want that to be translatable (I assume yes).